tests/baselines/reference/jsxExcessPropsAndAssignability.errors.txt

```diff
@@ -17,11 +17,11 @@ tests/cases/compiler/jsxExcessPropsAndAssignability.tsx(16,27): error TS2698: Sp
     
         const props: ComposedComponentProps = null as any;
     
-        // Expected no error, got none - good
+        // Expected no error caused by "myProp", got none - good

```

Comment:
I think in https://github.com/microsoft/TypeScript/pull/48570#discussion_r871930743 they were saying that because this comment says there's no error (i.e. "got none"), it's no longer true.

P.S. this is why I think comments like these are kind of silly in test cases.

Reply:
Can I remove them?

Reply:
I would be fine with that personally.

Reply:
Removed